--- SaWMan-1.6.2/src/sawman_config.c.orig	2012-09-20 17:19:17.000000000 +0200
+++ SaWMan-1.6.2/src/sawman_config.c	2012-10-06 19:28:41.470170130 +0200
@@ -85,7 +85,7 @@
                *next++ = '\0';
 
           if (strcmp (buf, "help") == 0) {
-               fprintf( stderr, config_usage );
+               fputs( config_usage, stderr );
                exit(1);
           }
 
@@ -515,7 +515,7 @@
           for (i = 1; i < *argc; i++) {
 
                if (!strcmp( (*argv)[i], "--sawman-help" )) {
-                    fprintf( stderr, config_usage );
+                    fputs( config_usage, stderr );
                     exit(1);
                }
 
